### DISCUSSION TITLE
How to create a fixed window

### DISCUSSION THREAD

**Discussion by sgj123456** [hyprwm/Hyprland#2903] [Q&A] [answered]
I want to create a window that remains fixed on the screen even when you switch workspaces. I tried to add the "pin" rule to the floating window, but it didn't take effect

**Comment by MightyPlaza** [comment, marked as answer]
pin window rule is only applied when window is 1st opened
if you want to make it pinned you probably want to use the `pin` dispatcher

**Reply by sgj123456** [reply]
`windowrulev2` = pin,class:^(kitty)$
Not effective

**Reply by sgj123456** [reply]
How should I set it up correctly

**Reply by MightyPlaza** [reply]
you have to also set a rule to make it float, use hyprctl dispatch pin or a keybind to make a window pinned

**Reply by sgj123456** [reply]
Success, thank you very much
